views/js/qtiCreator/widgets/helpers/formElement.js

```diff
@@ -150,7 +150,7 @@ define([
         const numericValue = options.floatVal ? parseFloat(value) : parseInt(value, 10);
         let isActualNumber = !isNaN(numericValue);
 
-        if (!options.allowNull && (numericValue === 0 || !isActualNumber)) {
+        if (!options.allowNull && !isActualNumber) {
```

Comment:
Add OrderInteraction. In side panel, switch radio-button to _"sort from source to target list_" . Preview --> there's `minChoices: 0` in `getItem` response. In previous version, there was neither `minChoices` property.

It should be fine, because `minChoices=0` and no defined `minChoices` have the same meaning, but still, if it's easy to keep previous behavior, maybe you should. If not, let's ignore this.